docs/template_plugin/src/template_executable_network.cpp

```diff
@@ -36,13 +37,49 @@ TemplatePlugin::ExecutableNetwork::ExecutableNetwork(const std::shared_ptr<const
 // ! [executable_network:ctor_cnnnetwork]
 
 // ! [executable_network:ctor_import_stream]
-TemplatePlugin::ExecutableNetwork::ExecutableNetwork(std::istream &                 model,
-                                                     const Configuration&           cfg,
-                                                     const Plugin::Ptr&             plugin) :
+TemplatePlugin::ExecutableNetwork::ExecutableNetwork(std::istream &       model,
```

Comment:
I didn't find import/export methods for template plugin, maybe I miss something?

Reply:
Import: https://github.com/openvinotoolkit/openvino/blob/b8a48b8a3d162879e89252fd40b0b82337d6209d/docs/template_plugin/src/template_plugin.cpp#L114-L128

Export:
https://github.com/openvinotoolkit/openvino/blob/b8a48b8a3d162879e89252fd40b0b82337d6209d/docs/template_plugin/src/template_executable_network.cpp#L190-L210